web/src/components/DisputeCard/index.tsx

```diff
@@ -67,7 +67,7 @@ const ListTitle = styled.div`
   height: 100%;
   justify-content: start;
   align-items: center;
-  width: ${responsiveSize(240, 300, 900)};
+  width: ${responsiveSize(240, 420, 900)};
```

Comment:
I get small cards with a lot of wasted space when the display is thinner 🤔 we could go with a grid based solution, similar to what we are doing on the Escrow: https://github.com/kleros/escrow-v2/blob/master/web/src/components/TransactionsDisplay/TransactionsGrid.tsx#L15
![image](https://github.com/kleros/kleros-v2/assets/80388279/5fd36ed4-5656-4fff-931c-f9afb9d5ba50)